### ISSUE TITLE
Port Editor does not update Clickable Ports, but YAML editor does

### ISSUE THREAD

**Issue by mrstux** [louislam/dockge#676]
### ⚠️ Please verify that this bug has NOT been reported before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### Description

Dockge has clickable links to exposed ports

<img width="203" alt="Screenshot 2024-11-28 at 3 26 05 PM" src="https://github.com/user-attachments/assets/1a633876-2eda-4bca-9cf1-2230be5b1199">

When using the Dockge port editor to add ports, eg:

<img width="675" alt="Screenshot 2024-11-28 at 3 27 19 PM" src="https://github.com/user-attachments/assets/15965a3a-faf9-4c6d-b7b6-45137a66923a">

The clickable ports are not updated when clicking save.

eg:

<img width="203" alt="Screenshot 2024-11-28 at 3 26 05 PM" src="https://github.com/user-attachments/assets/6cd2a375-58a9-4a4d-8ffe-c718df2f5318">

If instead, the YAML is edited, then they are
<img width="522" alt="Screenshot 2024-11-28 at 3 28 26 PM" src="https://github.com/user-attachments/assets/4f24efec-5d2a-468d-b0d0-c7a84eb29cb1">

<img width="316" alt="Screenshot 2024-11-28 at 3 28 38 PM" src="https://github.com/user-attachments/assets/e1fcfd34-719b-4b39-aeb4-bc02ea178c21">

### 👟 Reproduction steps

Using Dockge 1.4.2

Click "+ Compose", name the default stack, then deploy it

Then click "Edit"

Then in the stack editor, click Edit again

<img width="286" alt="Screenshot 2024-11-28 at 3 32 10 PM" src="https://github.com/user-attachments/assets/ef7803c0-7985-465e-88f0-c6ad39f37a36">

Change the port. Then click save.

The port will not change, see below, YAML is updated, clickable exposed port is not.

<img width="1147" alt="Screenshot 2024-11-28 at 3 33 16 PM" src="https://github.com/user-attachments/assets/6959d56a-0cc9-4ca2-9940-f17d58f9616c">

Click edit again, then change the port in the YAML... then click save. The port will change,

<img width="1091" alt="Screenshot 2024-11-28 at 3 33 42 PM" src="https://github.com/user-attachments/assets/9aa8e560-6db2-4626-bf5f-90d89677b322">

### 👀 Expected behavior

Changes to ports in the port edit view are not reflected by the UI

### 😓 Actual Behavior

Changes to ports in the port edit view should be reflected by the UI

### Dockge Version

1.4.2

### 💻 Operating System and Arch

TrueNAS Scale 24.10.0.2

### 🌐 Browser

Safari Version 17.6 (19618.3.11.11.5)

### 🐋 Docker Version

Docker version 27.1.1, build 6312585

**Comment by cyril59310** [comment]
You need to click on 'Deploy' to update the Compose and not on 'Save'. By clicking on 'Deploy', your ports will be updated

**Comment by mrstux** [comment]
Yes, the ports are updated in the yaml, but they are not updated in the UI, even though the application is launched with the correct ports, even if you use “deploy”
